BurnerB/FreeMentors-V2
# #168363445 User signup databases

#### What does this PR do?
Enables a user to signup

#### Description of Task to be completed
- Create users table
- create users model and controller
- ensure tests pass

#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ft-User-signup-DB-168363445
* Run npm --init to install all dependancies.
* Run command npm test on your terminal
* Tests should pass

#### What are the relevant pivotal tracker stories?
[#168363445](https://www.pivotaltracker.com/story/show/168363445)

#### Screenshots (if appropriate)
![signupdb](https://user-images.githubusercontent.com/45785786/64494271-6bac7300-d293-11e9-917f-a78e3639d53d.png)
